proto-lens-protoc/app/Data/ProtoLens/Compiler/Generate.hs

```diff
@@ -140,6 +141,17 @@ generateModule modName imports publicImports definitions importedEnv services
         , info <- allMessageFields env m
         , inst <- recordFieldLenses info
         ]
+    -- The packedFileDescriptorProto is a file level definition that's
+    -- shared across all message definitions.  If there are no
+    -- messages, it's omitted.
```

Comment:
Maybe clarify comment: it's omitted because it's not used?  (It took me a second to figure this out)